app/schema/portal/common/single-submit.js

```diff
@@ -1,8 +1,16 @@
 const Joi = require('joi')
 
-const confirmFlowValidFields = (field) => {
+const confirmFlowValidFields = (field, fields = []) => {
+  const moreFields = fields.reduce((base, newField) => {
```

Comment:
If you're trying to allow extra field names to not break the validation, I think you can just add '.unknown()' or something to the schema